README.md

```diff
@@ -30,84 +30,25 @@ We require node version 14.x.
 
 We are using **Netlify** to serve the docs in production.
 
-**Important:** `master` branch is automatically deployed to production.
+<!-- **Important:** `master` branch is automatically deployed to production. -->
 
 You can simulate the production build process locally by running.
 
 1. Run `npm run build`
 2. Run `npm start`
 
-## Feature branch / Deploy Preview
+## Feature branch / Deploy preview
 
-In order to test your work before merging to master, **Netlify** provides **Deploy Preview**. When raising a PR, github will provide **Netlify** notifications with the **Deploy Preview** url.
-
-## Versioning
```

Comment:
Is there a particular reason why we remove it completely? Maybe we could link the docs with versioning here? 🤔

Reply:
When I think about it, I don't know why someone might need to know the versioning of some of our APIs while reading README. The versioning is mentioned and explained several times in the docs, and the README is mostly about what this repo is, what the PLatform is, what Developer Program is, what you can build, how you can run the repository, so instructions on how to do things with the repository

Reply:
Okay, that makes a total sense😊